cuda enabled: True
action shape: (6,), prod shape: 6
Traceback (most recent call last):
  File "/home/icaros/anaconda3/envs/ppga/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/icaros/anaconda3/envs/ppga/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/icaros/Documents/PPGADev/RL/train_ppo.py", line 196, in <module>
    alg.train(vec_env, num_updates, rollout_length=cfg.rollout_length)
  File "/home/icaros/Documents/PPGADev/RL/ppo.py", line 299, in train
    self.next_obs = self.vec_inference.vec_normalize_obs(self.next_obs)
  File "/home/icaros/Documents/PPGADev/models/vectorized.py", line 130, in vec_normalize_obs
    obs[i] = normalizer(model_obs)
  File "/home/icaros/anaconda3/envs/ppga/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/icaros/Documents/PPGADev/utils/normalize.py", line 60, in forward
    obs = self.normalize(obs)
  File "/home/icaros/Documents/PPGADev/utils/normalize.py", line 65, in normalize
    obs = obs.to(self.obs_rms.mean.device)
AttributeError: 'numpy.ndarray' object has no attribute 'to'
Traceback (most recent call last):
  File "/home/icaros/anaconda3/envs/ppga/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/icaros/anaconda3/envs/ppga/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/icaros/Documents/PPGADev/RL/train_ppo.py", line 196, in <module>
    alg.train(vec_env, num_updates, rollout_length=cfg.rollout_length)
  File "/home/icaros/Documents/PPGADev/RL/ppo.py", line 299, in train
    self.next_obs = self.vec_inference.vec_normalize_obs(self.next_obs)
  File "/home/icaros/Documents/PPGADev/models/vectorized.py", line 130, in vec_normalize_obs
    obs[i] = normalizer(model_obs)
  File "/home/icaros/anaconda3/envs/ppga/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1190, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/icaros/Documents/PPGADev/utils/normalize.py", line 60, in forward
    obs = self.normalize(obs)
  File "/home/icaros/Documents/PPGADev/utils/normalize.py", line 65, in normalize
    obs = obs.to(self.obs_rms.mean.device)
AttributeError: 'numpy.ndarray' object has no attribute 'to'